lib/routes/actionsController.js

```diff
@@ -51,6 +54,12 @@ function PostAction(req, resp) {
     //Don't wait until action is finished
     resp.send({error: null, data: null});
 
+    if (config.isMaster) {
+        delay = 0;
+    }
+    else {
+        delay = 500;
```

Comment:
Not sure what this timeout is but if "1000" means a time, probably is better to have it unhardwired.